client-course-schedulizer/src/components/Tabs/FacultyLoads/FacultyLoads.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+import React, { useContext, useMemo } from "react";
+import { Column, useTable } from "react-table";
+import { Schedule, Term } from "../../../utilities/interfaces/dataInterfaces";
+import { ScheduleContext } from "../../../utilities/services/context";
+
+interface TableData {
+  faculty: string;
+  fallCourseSections?: string;
+  fallHours?: number;
+  loadNotes?: string;
+  otherDuties?: string;
+  otherHours?: number;
+  springCourseSections?: string;
+  springHours?: number;
+  summerCourseSections?: string;
+  summerHours?: number;
+  totalHours?: number;
+}
+
+const createTable = (schedule: Schedule): TableData[] => {
```

Comment:
Perhaps make a better name for the interface since it is an array? I guess I'm just a little confused as to if it is an individual row of data or data for multiple tables.

Reply:
It's an individual row so I'll change the name to TableRow